exercises/practice/space-age/.meta/tests.toml

```diff
@@ -32,3 +32,6 @@ description = "age on Uranus"
 
 [80096d30-a0d4-4449-903e-a381178355d8]
 description = "age on Neptune"
+
+[57b96e2a-1178-40b7-b34d-f3c9c34e4bf4]
+description = "invalid planet causes error"
```

Comment:
I'd rather not implement this, as it makes the exercise a bit harder. Could you not include this test?